contents/handbook/people/spending-money.md

```diff
@@ -50,18 +50,20 @@ PostHog will provide you with office equipment. Please note that it remains Post
 
 ### Laptop, Monitor, and Accessories
 
-We'd prefer you to use a laptop. This is so when we host meetups in real life, you can easily bring your work with you.
+We'd prefer you to use a laptop. This is so when we host meetups in real life, you can easily bring your work with you. 
 
-Our general laptop of choice is a MacBook Pro. Please look for a recent refurbished model. We'd prefer everyone uses Apple laptops, just to keep life simpler - for example, that means everyone can use the same software, and as we get bigger, it'll mean we're dealing with one supplier, not many.
+Our general laptop of choice is a MacBook Pro. Please look for something with an M1 chip or a recent refurbished model. We'd prefer everyone uses Apple laptops, just to keep life simpler - for example, that means everyone can use the same software, and as we get bigger, it'll mean we're dealing with one supplier, not many.
```

Comment:
I actually think for non-tech folks an Air with an M1 chip might be the right choice (it's still super fast and a bit lighter).

Reply:
I agree with Tim here.

With the introduction of Apple Silicon (M1), the Air makes a lot of sense. I would not buy any intel-based mac anymore (refurbished or not), as those are **a lot** slower and noisier than the M1 models, and will be phased out in the next year or two anyway.

Since the only Apple Silicon Macs right now are both 13", I'd recommend the Air. For both tech and non-tech users. It's lighter, $200 cheaper and comes without that useless touch bar. It also as fast as the "Pro", at least in the current lineup.

Reply:
Also, I recommend calling it "Apple Silicon" instead of the techier "M1", which will need to be updated twice per year as they release new chips.